aries-site/src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  Heading,
+  Layer,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const Feedback = ({
+  children,
+  modal,
+  onChange,
+  onClose,
+  onEsc,
+  onSubmit,
+  onReset,
+  show,
+  title,
+  value,
+  isSucessful,
+}) => {
+  const size = useContext(ResponsiveContext);
+
+  let content = (
+    <Box
+      fill="vertical"
+      overflow="auto"
+      width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+      pad="medium"
+      flex
+    >
+      <Identifier onClose={onClose} title={title} modal={modal} />
+      <Box gap="medium" margin={{ vertical: 'small' }}>
+        <Form
+          value={value}
+          onChange={onChange}
+          onSubmit={onSubmit}
+          onReset={onReset}
+          method="post"
+          validate="submit"
+        >
+          {children}
+          <>
+            {!isSucessful ? (
+              <Box
+                pad={{ top: 'medium' }}
+                justify="end"
+                gap="medium"
+                direction="row"
+              >
+                <Button onClick={onClose} label="Cancel" a11yTitle="Cancel feedback submission" />
+                <Button
+                  onSubmit={onSubmit}
+                  label="Submit"
+                  primary
+                  type="submit"
+                />
+              </Box>
+            ) : (
+              <Box align="end" pad={{ top: 'medium', bottom: 'small' }}>
+                <Text alignSelf="end" weight="bold">
+                  Thank You!
+                </Text>
+              </Box>
+            )}
+          </>
```

Comment:
Trying to minimize the number of props we are using and have Boxes be internally focused on spacing and avoid margins.

```suggestion
<Box gap="medium">
          <>{children}</>
          <Box direction="row" justify="end" gap="medium">
            {!isSucessful ? (
              <Box direction="row" gap="small">
                <Button
                  onClick={onClose}
                  label="Cancel"
                  a11yTitle="Cancel feedback submission"
                />
                <Button
                  onSubmit={onSubmit}
                  label="Submit"
                  primary
                  type="submit"
                />
              </Box>
            ) : (
              <Text alignSelf="end" weight="bold">
                Thank You!
              </Text>
            )}
          </Box>
        </Box>
```